lightning/web-app.md

```diff
@@ -147,7 +147,7 @@ $ sudo su - thunderhub
 * Set a temporary version environment variable to the installation
 
 ```bash
-$ VERSION=0.13.20
+$ VERSION=0.13.21
```

Comment:
Thank you for your contribution. Did you test the update process and correct execution after the update?

Reply:
Yes, I did.

Reply:
I had a problem after the update. It automatically logs me out when I log in, not being able to use the service.
I tried using another browser, in private mode, reinstalling ThunderHub, etc., but just downgrading to v0.13.20 again, fixed it.

Didn't that happen to you?

Reply:
Here version 0.13.21 is working with no issues. Anyway, in my case it was a fresh install, not an update. I don't know if, for example, some user data could affect it in your case.

Reply:
> Here version 0.13.21 is working with no issues. Anyway, in my case it was a fresh install, not an update. I don't know if, for example, some user data could affect it in your case.

I did a complete reinstalling and the issue persisted, just downgrading to v0.13.20 again, fixed it.